topics/uptime-results.yaml

```diff
@@ -0,0 +1,20 @@
+topic: uptime-results
+pipeline: uptime
+description: uptime check results
+services:
+  producers:
+    - getsentry/uptime-checker
+  consumers:
+    - getsentry/sentry
+schemas:
+  - version: 1
+    compatibility_mode: none
+    type: json
+    resource: uptime-results.v1.schema.json
+    examples:
+      - uptime-results/1/
+topic_creation_config:
+  compression.type: lz4
+  message.timestamp.type: LogAppendTime
+  retention.ms: "86400000"
+partitions: 64
```

Comment:
Please remove this line. It's only intended to capture topics where partitions MUST be fixed to 1. Though I see how confusing this is and I'll add some proper validation around this as a follow up.

Reply:
ah ok. Maybe we can rename it to `enforced_partition_count` or something similar 👍

Reply:
yeah, i like that name